src/EditorFeatures/Core/Diagnostics/DiagnosticsSquiggleTaggerProvider.cs

```diff
@@ -126,5 +127,12 @@ protected internal override bool IncludeDiagnostic(DiagnosticData diagnostic)
                     return PredefinedErrorTypeNames.OtherError;
             }
         }
+
+        protected override bool Equals(IErrorTag tag1, IErrorTag tag2)
+        {
+            return tag1 is RoslynErrorTag errorTag1 &&
```

Comment:
Should this do a hard cast, since we don't expect error tags to not be a RoslynErrorTag?

Reply:
yes.  that seems reasonable.